fdbclient/FDBTypes.h

```diff
@@ -737,6 +737,7 @@ struct HealthMetrics {
 	int64_t worstStorageDurabilityLag;
 	int64_t worstTLogQueue;
 	double tpsLimit;
+	bool batchLimited;
```

Comment:
About the commit message, could you describe why DD needs this information?

Reply:
Sure. I'll add the following:

DD uses this information to determine when to do the rebalance in the background, i.e., moving data from heavily loaded servers to lighter ones. If the cluster is currently rate limited for batch commits, then the rebalance will use longer time intervals, otherwise use shorter intervals.